test/suite/test_timestamp14.py

```diff
@@ -259,7 +259,7 @@ def test_all_durable(self):
         # We have a running transaction with a lower commit_timestamp than we've
         # seen before. So all_durable should return (lowest commit timestamp - 1).
         session1.begin_transaction()
-        cur1[1] = 2
+        cur1[2] = 2
```

Comment:
This test was committing out of order. It's only to show that `all_durable` doesn't move backwards so it's fine to just insert on separate keys.